src/ConvertInputFormat/main.cpp

```diff
@@ -145,8 +145,12 @@ void cleanEPJSON(json &epjson)
     }
 }
 
-bool processInput(
-    std::string const &inputFilePath, json const &schema, OutputTypes outputType, std::string outputDirectory, std::string &outputTypeStr)
+bool processInput(std::string const &inputFilePath,
+                  json const &schema,
+                  OutputTypes outputType,
+                  std::string outputDirectory,
+                  std::string &outputTypeStr,
+                  bool convertHVACTemplate)
```

Comment:
Add parameter to control whether HVACTemplate objects are converted or rejected.

Reply:
OK.  I'm curious how this parameter is assigned in the calling routine.